src/index.ts

```diff
@@ -28,25 +29,35 @@ async function run() {
       workflowId,
       ARTIFACT_NAME,
       FILE_NAME
-    )) || { sha: 'none', data: [] };
+    )) || { sha: 'none', data: {route: [], dynamicChunks: []}};
     console.log(masterBundleSizes);
 
     console.log('> Calculating local bundle sizes');
-    const bundleSizes = getBundleSizes();
+    const bundleSizes = getStaticBundleSizes();
     console.log(bundleSizes);
+    const dynamicBundleSizes = getDynamicBundleSizes();
+    console.log(dynamicBundleSizes);
 
     console.log('> Uploading local bundle sizes');
-    await uploadJsonAsArtifact(ARTIFACT_NAME, FILE_NAME, bundleSizes);
+    await uploadJsonAsArtifact(ARTIFACT_NAME, FILE_NAME, {
```

Comment:
If you uploaded the dynamic bundle sizes in a different file, then it wouldn't break against existing master builds.

Reply:
done. works as expected: 

https://github.com/transferwise/homepage/pull/1109#issuecomment-961964880
https://github.com/transferwise/balance-flows-web/pull/817#issuecomment-962966643